kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/ScaleOffsetType.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eurotech and/or its affiliates and others
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *  Eurotech
+ ******************************************************************************/
+package org.eclipse.kura.web.shared;
+
+/**
+ * This contains all the required type constants for representing
+ * Scale and Offset.
+ * 
+ * This is a copy of {@code org.eclipse.kura.channel.ScaleOffsetType}
+ *
```

Comment:
Same question here.

Reply:
Because we need it in the client part of webui